botocore/data/_retry.json

```diff
@@ -84,6 +84,16 @@
           "too_many_requests": {"$ref": "too_many_requests"}
       }
     },
+    "apigateway": {
+      "__default__": {
```

Comment:
This changes the strategy for every apigateway operation, not just the operations specifically impacted.